diff --git a/node_modules/rollup-plugin-dts/dist/rollup-plugin-dts.cjs b/node_modules/rollup-plugin-dts/dist/rollup-plugin-dts.cjs
index bd2f2d9..936f0ba 100644
--- a/node_modules/rollup-plugin-dts/dist/rollup-plugin-dts.cjs
+++ b/node_modules/rollup-plugin-dts/dist/rollup-plugin-dts.cjs
@@ -51,8 +51,6 @@ const OPTIONS_OVERRIDE = {
     checkJs: false,
     declarationMap: false,
     skipLibCheck: true,
-    // Ensure TS2742 errors are visible
-    preserveSymlinks: true,
     // Ensure we can parse the latest code
     target: ts__default['default'].ScriptTarget.ESNext,
 };
